README.md

```diff
@@ -817,6 +818,84 @@ be named secretless.
     ```
     "test_app:84674b2874a5d7c952e7fec8"
     ```
+   
+### Deploy Secrets Provider Sidecar
+
+1. Deploy an app with the Secrets Provider Sidecar:
+
+   The **Secrets Provider Sidecar** is injected into the application pod on pod creation via the
+   sidecar injector. The injection is configured via annotations.
+
+ + The `conjur.org/container-name` is set to "cyberark-secrets-provider-for-k8s".
+
+    ```bash
+    ~$ kubectl -n ${TEST_APP_NAMESPACE_NAME} \
+      delete pod \
+      test-app --ignore-not-found
+   
+    ~$ cat << EOF | kubectl -n ${TEST_APP_NAMESPACE_NAME} apply -f -
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      annotations:
+        conjur.org/inject: "true"
+        conjur.org/inject-type: "secrets-provider"
+        conjur.org/container-name: "cyberark-secrets-provider-for-k8s"
+        conjur.org/container-image: "docker.io//secrets-provider-for-k8s:edge"
```

Comment:
is it intended to have an empty string in this container-image path? i.e. in between docker.io and secrets-provider-fork8s:edge

"docker.io/ /secrets-provider-for-k8s:edge"

Reply:
no, thanks for finding it.